core/CronArchive.php

```diff
@@ -1873,6 +1901,7 @@ private function makeCliMulti()
         $cliMulti->setUrlToPiwik($this->urlToPiwik);
         $cliMulti->setPhpCliConfigurationOptions($this->phpCliConfigurationOptions);
         $cliMulti->setAcceptInvalidSSLCertificate($this->acceptInvalidSSLCertificate);
+        $cliMulti->setConcurrentProcessesLimit($this->getConcurrentRequestsPerWebsite());
```

Comment:
This change was in theory not needed because we only set multiple URLs where it was already set, but ensures we always respect max concurrent requests is being respected.